jornet-server/src/domains/score.rs

```diff
@@ -105,6 +112,7 @@ pub(crate) fn score() -> impl HttpServiceFactory {
         .wrap(cors)
         .route("{leaderboard_id}", web::post().to(save_score))
         .route("{leaderboard_id}", web::get().to(get_scores))
+        .route("{leaderboard_id}", web::delete().to(delete_scores))
```

Comment:
putting the new endpoint here makes it reachable without authentication

You would need to put it in a scope that checks the bearer (for example https://github.com/vleue/jornet/blob/main/jornet-server/src/domains/leaderboard.rs#L58), and check that the leaderboard owner is the same as the token user